case_studies/strong_lensing/galsim-random.yaml

```diff
@@ -0,0 +1,79 @@
+
+variables :
+  output_directory : data/images
+  n1 : 2.5
+  half_light_radius : 2
+  flux1 : .3
+  n2 : 1.5
+  scale_radius : 1
+  flux2 : .7
+  q : .7
+  beta : 23
+
+# Define the galaxy profile
+gal :
+    type : Sum
+    items :
+        -   type : Sersic
+            n : '@variables.n1'
+            half_light_radius : '@variables.half_light_radius'
+            flux : '@variables.flux1'
+        -   type : Sersic
+            n : '@variables.n2'
+            scale_radius : '@variables.scale_radius'
+            flux : '@variables.flux2'
+  
+    shear :
+        type : QBeta
+        q : '@variables.q'
+        beta : '@variables.beta'
+
+    flux : 1.e6
+
+# Define the PSF profile
+psf :
+    type : Convolution
+    items :
+        -
+            type : Kolmogorov
```

Comment:
Is this right PSF for DES? There's a dedicated DES PSF class called `DES_PSFEx`:
https://galsim-developers.github.io/GalSim/_build/html/des.html#des-psf-models

Reply:
That's what there using here, are you modeling your config on a different file?
https://github.com/GalSim-developers/GalSim/blob/releases/2.5/examples/des/blendset.yaml

Reply:
I have changed it to this DES psf in the latest commit.